controllers/gslb_controller.go

```diff
@@ -84,11 +84,13 @@ func (r *GslbReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.
 			// Return and don't requeue
 			return result.Stop()
 		}
+		m.ErrorIncrement(gslb)
 		return result.RequeueError(fmt.Errorf("error reading the object (%s)", err))
 	}
 
 	err = r.DepResolver.ResolveGslbSpec(ctx, gslb, r.Client)
 	if err != nil {
+		m.ErrorIncrement(gslb)
```

Comment:
Consider using VerbNoun naming for the function in order to retain the consistency with other functions in the code (e.g. `RequeError()`):
```suggestion
		m.IncrementError(gslb)
```

Reply:
@somaritane , than would be nice to follow pattern and rename more than that. WDYT ? 

ErrorIncrement -> IncrementError
ReconciliationIncrement -> IncrementReconciliation
InfobloxZoneUpdateIncrement -> InfobloxIncrementZoneUpdate
InfobloxZoneUpdateErrorIncrement -> InfobloxIncrementZoneUpdateError
InfobloxHeartbeatIncrement -> InfobloxIncrementHeartbeat
InfobloxHeartbeatErrorIncrement -> InfobloxIncrementHeartbeatError

Reply:
@somaritane , amended for all increment funcs